ocs_ci/helpers/cnv_helpers.py

```diff
@@ -365,3 +365,48 @@ def run_dd_io(vm_obj, file_path, size="10240", username=None, verify=False):
             file_path=file_path,
             username=username,
         )
+
+
+def expand_pvc_and_verify(vm_obj, new_size, failed_vms, vm_objs_def=None):
```

Comment:
why do you need an arg to store the failed VM names?

Reply:
IMO, it should not be used as an arg and it has nothing to do with the pvc expansion